src/Sarif.Converters/ToolFormat.cs

```diff
@@ -4,27 +4,36 @@
 namespace Microsoft.CodeAnalysis.Sarif.Converters
 {
     /// <summary>Values referring to a source format to convert to the static analysis results interchange format.</summary>
-    public enum ToolFormat
+    public static class ToolFormat
```

Comment:
>ToolFormat [](start = 24, length = 10)

as per other mail, suggest adding a simple test that puts all ToolFormat members into a case-insensitive dictionary or set, ensuring no member is already present. would use reflection for this to keep the test maintenance free as type changes.